README.md

```diff
@@ -0,0 +1,65 @@
+# SSS TODO_track_name
+
+This is the public repository for the **TODO** track of the Security Summer School (SSS).
+It contains information available to participants: session content, slides, activities support files.
+Apart from SSS participants, you can use this content by yourself to learn more about **TODO**.
+
+## Using the Content
+
+As with all SSS tracks, the **TODO** track consists of multiple sessions.
+One session typically covers a particular topic of the track.
+Ideally you would go through sessions orderly, but you can jump ahead to a topic of interest.
+
+Each session has its own folder.
+Access the `README.md` file in each session folder.
```

Comment:
```suggestion
The `README.md` file from a session folder details the topic of the session and includes practical demos of tools and snippets of code.
```
(Or something like that - I didn't like the short, repetitive, separate sentences.)